src/mudlet.cpp

```diff
@@ -2341,16 +2352,21 @@ void mudlet::slot_notes()
 
 void mudlet::slot_irc()
 {
-    if( ! mpIRC )
-    {
-        mpIRC = new dlgIRC();
-        mpIRC->setWindowTitle( tr( "Mudlet live IRC Help Channel #mudlet-help on irc.freenode.net" ) );
-        mpIRC->setWindowIcon( QIcon( QStringLiteral( ":/icons/mudlet_irc.png" ) ) );
-        mpIRC->resize(660,380);
+    Host * pHost = getActiveHost();
+    bool isDefaultHost = false;
+    if( ! pHost ) {
+        pHost = mpDefaultHost;
+        isDefaultHost = true;
     }
 
-    mpIRC->raise();
-    mpIRC->show();
+    if( ! mpIrcClientMap.contains(pHost) )
+    {
+        QPointer<dlgIRC> dlg = new dlgIRC(pHost);
+        dlg->setDefaultHostClient(isDefaultHost);
+        mpIrcClientMap[pHost] = dlg;
```

Comment:
I forgot to update the calls in `TLuaInterpreter.cpp` which was likely the culprit of the crash.